src/main/java/org/folio/rest/utils/PatronNoticeBuilder.java

```diff
@@ -235,6 +243,12 @@ private static String getAllContributors(Instance instance) {
       .collect(joining(LIST_VALUES_SEPARATOR));
   }
 
+  private static String getDatesOfPublication(Instance instance) {
+    return instance.getPublication().stream()
```

Comment:
Will return value of `instance.getPublication()` always be non-null?

Reply:
Yes.